README.md

```diff
@@ -1,15 +1,89 @@
-# <div align="left"><img src="img/rapids_logo.png" width="90px"/>&nbsp;cuGraph - GPU Graph Analytics</div>
+<h1 align="center"; style="font-style: italic";>
+  <br>
+  <img src="img/cugraph_logo_2.png" alt="cuGraph" width="500">
+</h1>
+
+<div align="center">
 
 [![Build Status](https://gpuci.gpuopenanalytics.com/job/rapidsai/job/gpuci/job/cugraph/job/branches/job/cugraph-branch-pipeline/badge/icon)](https://gpuci.gpuopenanalytics.com/job/rapidsai/job/gpuci/job/cugraph/job/branches/job/cugraph-branch-pipeline/)
+<a href="https://github.com/rapidsai/cugraph/blob/main/LICENSE">
+    <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" alt="License"></a>
+<img alt="GitHub tag (latest by date)" src="https://img.shields.io/github/v/tag/rapidsai/cugraph">
+
+<a href="https://github.com/rapidsai/cugraph/stargazers">
+    <img src="https://img.shields.io/github/stars/rapidsai/cugraph"></a>
+<img alt="Conda" src="https://img.shields.io/conda/dn/rapidsai/cugraph">
+<img alt="GitHub last commit" src="https://img.shields.io/github/last-commit/rapidsai/cugraph">
+
+<img alt="Conda" src="https://img.shields.io/conda/pn/rapidsai/cugraph" />
+
+<a href="https://rapids.ai/"><img src="img/rapids_logo.png" alt="RAPIDS" width="125"></a>
+
+</div>
+
+<br>
+
+[RAPIDS](https://rapids.ai) cuGraph is a monorepo that represents a collection of packages focused on GPU-accelerated graph analytics, including support for property graphs, remote (graph as a service) operations, and graph neural networks (GNNs).  cuGraph supports the creation and manipulation of graphs followed by the execution of scalable fast graph algorithms.
+
+<div align="center">
+
+[Getting cuGraph](./readme_pages/getting_cugraph.md) *
+[Graph Algorithms](./readme_pages/algorithms.md) *
+[Graph Service](./readme_pages/cugraph_service.md) *
+[Property Graph](./readme_pages/property_graph.md) *
+[GNN Support](./readme_pages/gnn_support.md)
+
+</div>
+
+-----
+
+## Table of content
+- Getting packages
+  - [Getting cuGraph Packages](./readme_pages/getting_cugraph.md)
+  - [Contributing to cuGraph](./readme_pages/CONTRIBUTING.md)
+- General
+  - [Latest News](./readme_pages/news.md)
+  - [Current list of algorithms](./readme_pages/algorithms.md)
+  - [BLOGs and Presentation](./docs/cugraph/source/basics/cugraph_blogs.rst)

```

Comment:
Can we 

1. reverse the order of the timeline, so newest stuff is first
2. add event notebooks